.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please `git rm` this file (or the whole directory).  It shouldn't be included in git.  You could add it to `.gitignore` so it's not tracked

Reply:
done